### ISSUE TITLE
Doesn't compile, missing files.

### ISSUE THREAD

**Issue by shujaatak** [sheepeoff/Qml-I-O#1]
I have Firebase C++ SDK, but many other files are missing in this repository. Can you please upload those files?
I am interested in that code which use Firebase. I am looking for understanding how to use Firebase C++ SDK in Qt.

**Comment by sheepeoff** [comment]
Yes and i know some files are missing. Have you tried qtfirebase because I currently have a beta version of the real working files (still testing)  but to be released by this month

**Comment by shujaatak** [comment]
I have tried Larpon/QtFirebase but I don't know why it's not working for me. I am using Qt5.14.2 and Firebase version 6.14.0 so both Qt and Firebase C++ SDK are latest.

**Comment by sheepeoff** [comment]
I had the same problem and i couldnot fix it because qt 5.14 uses only clang compiler for android 
i added these flags in my .pro file
--------------------------------------------
```
QMAKE_CXXFLAGS += -fno-exceptions -fno-rtti
QMAKE_LFLAGS -= -nostdlib++    
```
--------------------------------------------
but it didnot do any work.
Then i realised that i have to rebuild the firebase sdk using the qt 5.14 clang compiler which gave errors.
So i just downgraded to Qt 5.11 bacause it supports the gcc compiler and its working
